ion-schema/src/system.rs

```diff
@@ -181,14 +181,14 @@ impl PendingTypes {
             Some(id) => self.types_by_id[*id]
                 .to_owned()
                 .and_then(|type_def| match type_def {
-                    TypeDefinition::Named(named_type_def) => Some(Ok(named_type_def)),
-                    TypeDefinition::Anonymous(_) => {
+                    TypeDefinitionKind::Named(named_type_def) => Some(Ok(named_type_def)),
```

Comment:
All the changes in this file are related to renaming of `TypeDefinition` to `TypeDefinitionKind`.